ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -57,6 +57,40 @@ objects:
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
+    initContainers:
```

Comment:
I would have expected the following workflow:

 - define the bazel image as your `test_base_image` (or define one that adds bazel + whatever other deps you have) [doc](https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#test_base_image)
 - run the `bazel-build-smoke` commands in your `test_binary_build_commands` [doc](https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#test_binary_build_commands)
 - use the `$LOCAL_IMAGE_TEST_BIN` (`pipeline:test-bin`) as your image for a test container running before the e2e one

Reply:
Ok, taking that in.. I would have not gotten that workflow w/out you pointing those exact points in the docs out - I'm happy to do things in the intended way, and then pass on the info to others looking to utilize ci-operator. The bazel env is a bit tricky, and I was glad to get it working for us the way I did, but I'll reconfigure this PR to match your suggested workflow (nope, not annoyed at all.. lol).

Reply:
^^ btw,  'your suggested workflow' definitely makes more sense than the initContainers and is obviously the right way to do this :+1:  thanks

Reply:
Once you've workded through this let's work on making out doc better for the next person -- sorry about the state of our doc today ...